packages/vega-loader/package.json

```diff
@@ -20,6 +20,8 @@
   "author": "Jeffrey Heer (http://idl.cs.washington.edu)",
   "main": "build/vega-loader.node.js",
   "module": "index",
+  "unpkg": "build/vega-loader.min.js",
```

Comment:
Alternatively, you can add `"browser": build/vega-loader.min.`

Reply:
We already use browser in a more nuanced way, and in addition (according to vega/dataflow-api#3) Observable / d3-require doesn't use the browser field.

Reply:
It does but only after the other two. https://github.com/d3/d3-require/blob/master/src/index.js

Reply:
Yep—to clarify, d3-require *will* use the `browser` entry if it’s a string, but not if it’s an object.